suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml

```diff
@@ -149,94 +151,105 @@ tests:
       module: exec.py
       name: configure mon_allow_pool_delete to True
 
-  - test:
-      name: test-image-delete-from-primary-site
-      module: test-image-delete-primary-site.py
-      clusters:
-        ceph-rbd1:
-          config:
-            imagesize: 2G
-            io-total: 200M
-      polarion-id: CEPH-9501
-      desc: Verify that image deleted at primary site updated at secondary
+  # - test:
```

Comment:
same here reviewer feel scary to approve, make sure please raise PR which you feel like it will get mere please avoid commented test

Reply:
done.